api/README.md

```diff
@@ -45,6 +45,10 @@ browser names (`chrome`, `edge`, `firefox`, `safari`) and release channels (`exp
 or `stable`).
 
 __`from`__ : RFC3339 timestamp, for which to include runs that occured after the given time.
+NOTE: Runs are sorted by `time_start` descending, so be wary when combining this parameter
```

Comment:
This detail applies regardless of presence/value of `from`. How about extending the description of the data:

```diff
-Gets the TestRun metadata for all runs for a given SHA[0:10].
+Gets the TestRun metadata for all runs for a given SHA[0:10], sorted by `time_start` descending.

Reply:
Sorry, merged too fast, did this in a follow-up tweak though.

Reply:
Thanks!